lib/HLSL/DxilValidation.cpp

```diff
@@ -36,6 +36,7 @@
 #include "dxc/HLSL/DxilSpanAllocator.h"
 #include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/BitVector.h"
+#include "llvm/ADT/SmallSet.h"
```

Comment:
I think I added this during an intermediate version and forgot to remove it again.  But strangely, there are uses of SmallPtrSet, and no inclusion of `SmallPtrSet.h`, which `SmallSet.h` will also include.  I wonder where it's getting the necessary header from.  Must be from one of the other headers somewhere.  Should I just change this to `SmallPtrSet.h`, since that one is directly used in this file?